### ISSUE TITLE
batch-norm in classification layer?

### ISSUE THREAD

**Issue by mgarbade** [ildoonet/tf-pose-estimation#593]
Checking the mobilenet v2 architecture it seems like every separable_convolution  in the CMP part of the model is followed by a batch-norm:

for example [here](https://github.com/ildoonet/tf-pose-estimation/blob/master/tf_pose/network_mobilenet_v2.py#L79):

                (self.feed(prefix + '_concat')
                 # .se_block(name=prefix + '_L2_se', ratio=8)
                 .separable_conv(3, 3, depth2(128), 1, name=prefix + '_L2_1')
                 .separable_conv(3, 3, depth2(128), 1, name=prefix + '_L2_2')
                 .separable_conv(3, 3, depth2(128), 1, name=prefix + '_L2_3')
                 .separable_conv(1, 1, depth2(128), 1, name=prefix + '_L2_4')
                 .separable_conv(1, 1, 19, 1, relu=False, name=prefix + '_L2_5'))

which is called like [this](https://github.com/ildoonet/tf-pose-estimation/blob/9aaf34708e9863eeab9685a7ec52a45f3b576e2d/tf_pose/network_base.py#L206):

    def separable_conv(self, input, k_h, k_w, c_o, stride, name, relu=True, set_bias=True):
        with slim.arg_scope([slim.batch_norm], decay=0.999, fused=common.batchnorm_fused, is_training=self.trainable):
            output = slim.separable_convolution2d(input,
                                                  [...]
                                                  activation_fn=None,
                                                  # normalizer_fn=slim.batch_norm,
                                                  [...]
                                                  scope=name + '_depthwise')

            output = slim.convolution2d(output,
                                        [...]
                                        activation_fn=common.activation_fn if relu else None,
                                        [...]
                                        normalizer_fn=slim.batch_norm,
                                        [...]
                                        scope=name + '_pointwise')

        return output

Shouldn't the classification layers for heatmaps and PAFs be free of batch-norm?
